### ISSUE TITLE
Unstable behaviour at the level of wagtailadmin display

### ISSUE THREAD

**Issue by DhiaTN** [infoportugal/wagtail-modeltranslation#56]
When I add `TranslationMixin` to my translatable page, it adds always correctly the multi-language fields to the database, but it does always display them all in the Wagtail admin page. Let's consider the following code:

``` python
class HomePage(Page, TranslatablePageMixin):
    claim = models.CharField(
        verbose_name='claim',
        max_length=255,
        blank=True
    )

    header_image = models.ForeignKey(
        'wagtailimages.Image',
        null=True,
        blank=True,
        on_delete=models.SET_NULL,
        related_name='+'
    )


HomePage.content_panels = [
    FieldPanel('title', classname="full title"),
    FieldPanel('claim', classname="full title"),
    ImageChooserPanel('header_image')
]


class SubmitFormField(AbstractFormField):
    page = ParentalKey('SubmitFormPage', related_name='form_fields')


class SubmitFormPage(WagtailCaptchaEmailForm, TranslationMixin):
    body = RichTextField(blank=True, help_text='Edit the content you want to see before the form.')
    thank_you_text = RichTextField(blank=True, help_text='Set the message users will see after submitting the form.')


SubmitFormPage.content_panels = Page.content_panels + [
    MultiFieldPanel([
        FieldPanel('to_address'),
        FieldPanel('from_address'),
        FieldPanel('subject'),
    ], "Email"),
    FieldPanel('body', classname="full"),
    FieldPanel('thank_you_text', classname="full"),
    # InlinePanel('form_fields', label="Form fields")
]
```

If we consider that for both pages I added the translatable attributes to their correspondent Translation classes in `translation.py` the result is as follow:

**SubmitFormPage :** (works correctly)

![snapshot2](https://cloud.githubusercontent.com/assets/2914703/12642095/efc9e384-c5b3-11e5-8cc5-87349af119ca.png)

**HomePage :** (does not work as expected)

![snapshot1](https://cloud.githubusercontent.com/assets/2914703/12642102/f8437fe8-c5b3-11e5-9dc6-137a3f1a4855.png)

In order to fix the problem I had to rewrite the `HomePage.content_panels` and add all the fields manually which is not the way it is intended to be used:

``` python
HomePage.content_panels = [
    FieldPanel('title_en', classname="full title"),
    FieldPanel('title_de', classname="full title"),
    FieldPanel('claim_en', classname="full title"),
    FieldPanel('claim_de', classname="full title"),
    ImageChooserPanel('header_image')
]
```

**Comment by nikonikoniko** [comment]
I am also experiencing this same behaviour.  

```
wagtail-modeltranslation==0.4.2
wagtail==1.3.1
```

**Comment by Alexandre27** [comment]
Should be fixed in version 0.6. See upgrade considerations on readme if you wish to upgrade.
